libunwind/src/DwarfInstructions.hpp

```diff
@@ -105,6 +99,9 @@ class DwarfInstructions {
   static bool getRA_SIGN_STATE(A &addressSpace, R registers, pint_t cfa,
                                PrologInfo &prolog);
 #endif
+#if defined(__CHERI_PURE_CAPABILITY__) && defined(_LIBUNWIND_CHERI_C18N_SUPPORT)
+  static constexpr CompartmentInfo<A, R> CompartInfo{};
```

Comment:
You don't actually need an instance of it, everything's static methods. Just `typedef CompartmentInfo<A, R> C18N` or something?